### DISCUSSION TITLE
support OPA in Apache APISIX for access control

### DISCUSSION THREAD

**Discussion by juzhiyuan** [open-policy-agent/community#73] [Community]
Hi community, I noticed Apache APISIX is supporting the OPA plugin, and here has one proposal[1] 🙈 

Apache APISIX is a Cloud-Native API Management Platform, see https://apisix.apache.org/

Anyway, welcome to have a Proposal check and leave your comments 🥳

[1] https://github.com/apache/apisix/issues/5714

**Comment by anderseknert** [comment]
Great to see this @juzhiyuan ! Can't wait to have it added to the list of [integrations](https://www.openpolicyagent.org/docs/latest/ecosystem/) :)
Let me know if there's anything I can do to help support your integration from the OPA side of things.

**Reply by juzhiyuan** [reply]
Thanks, @anderseknert! Will let you know when need help 😄
